src/components/map.js

```diff
@@ -2,7 +2,10 @@ import React from "react";
 import PropTypes from "prop-types";
```

Comment:
Got it, was getting messy and figured I'd hit the file with it since I wasn't aware of a way to do it only for certain sections but I'll try to find something that is more surgical